CONTRIBUTING.md

```diff
@@ -107,7 +112,7 @@ Check out the [Server Requirements for Running Zen Cart](/user/first_steps/serve
 not this: 
 
 <pre>
-Check out the [Server Requirements for Running Zen Cart](/user/first_steps/server_requirements.md).  
+Check out the [Server Requirements for Running Zen Cart](/user/first_steps/server_requirements.md).        **NO!**
```

Comment:
???

Reply:
Means "don't do this" so people don't accidentally copy the bad line.   It was there for other "not this" cases.

Reply:
Ok. Got it. I'd missed which file this was in while scanning all the other changes. 😊